.github/workflows/yamllinter.yaml

```diff
@@ -0,0 +1,27 @@
+name: Yaml Lint
+on:
+  pull_request:
+    paths:
+    - '**.yaml'
+
+jobs:
+  yaml-validator-and-linter :
```

Comment:
wow lots of errors from the yaml linter!

Reply:
right, will take some time to resolve all ;)